lib/fastlane/plugin/wpmreleasetoolkit/actions/configure/configure_apply_action.rb

```diff
@@ -22,6 +33,12 @@ def self.run(params = {})
             end
         }
 
+        ### Restore secrets repo to original branch.  If it was originally in a 
+        ### detached HEAD state, we need to extract the hash from the branch name.
+        original_repo_branch = original_repo_branch.sub("(HEAD detached at ", "").sub(")", "")
```

Comment:
This seems a little brittle. Can we use `repo_hash` if the branch isn't present? Perhaps you could add logic to `Fastlane::Helper::ConfigureHelper` so that `repo_branch_name` is set to nil if its not a branch.

Reply:
> Perhaps you could add logic to Fastlane::Helper::ConfigureHelper so that repo_branch_name is set to nil if its not a branch.

That’s a great suggestion!  We could, in the helper, check to see if the branch name begins with `(HEAD detached at` and, if so, return `nil`.

Reply:
> check to see if the branch name begins with `(HEAD detached at` 
Yes, or even better, check that its on a branch. Something like [this](https://stackoverflow.com/questions/52221558/programmatically-check-if-head-is-detached) or [this](https://stackoverflow.com/questions/5167957/is-there-a-better-way-to-find-out-if-a-local-git-branch-exists) perhaps.

Reply:
That is super cool.  I haven’t used `git rev-parse` much and its usefulness is blowing my mind.  😄  I’ll look at those posts - thanks!